.github/workflows/main.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         run: |
           rustup set profile minimal
           rustup toolchain install nightly
+          rustup target add --toolchain nightly aarch64-apple-darwin
```

Comment:
A comment describing why this is needed would be great.

Reply:
Is d10e6af sufficient?

Reply:
Yup, thanks!